scripts/obwatch/ob-watcher.py

```diff
@@ -352,13 +352,13 @@ def create_fidelity_bond_table(self, btc_unit):
                 utxo_value_str = "No data"
             else:
                 bond_value_str = satoshi_to_unit_power(bond_value, 2*unit_to_power[btc_unit])
-                conf_time_str = str(datetime.utcfromtimestamp(conf_time))
+                conf_time_str = str(datetime.utcfromtimestamp(0) + timedelta(seconds=conf_time))
                 utxo_value_str = satoshi_to_unit(utxo_data["value"], None, btc_unit, 0)
             bondtable += ("<tr>"
                 + elem(bond_data.maker_nick)
                 + elem(bintohex(bond_data.utxo[0]) + ":" + str(bond_data.utxo[1]))
                 + elem(bond_value_str)
-                + elem(datetime.utcfromtimestamp(bond_data.locktime).strftime("%Y-%m-%d"))
+                + elem(datetime.utcfromtimestamp(0) + timedelta(seconds=bond_data.locktime).strftime("%Y-%m-%d"))
```

Comment:
this causes an error so will revert

Reply:
Add paranthesis? i.e. `(datetime.utcfromtimestamp(0) + timedelta(seconds=bond_data.locktime)).strftime("%Y-%m-%d")`

Reply:
without this change i.e. leaving it as 
`elem(datetime.utcfromtimestamp(bond_data.locktime).strftime("%Y-%m-%d"))`
the ob-watcher.py runs fine on my 32 bit machine as well.

Trying mine or the suggested change above leads to the error:
```
Exception happened during processing of request from ('127.0.0.1', 42698)
Traceback (most recent call last):
  File "/usr/lib/python3.7/socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python3.7/socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python3.7/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "./ob-watcher.py", line 204, in __init__
    directory=os.path.dirname(os.path.realpath(__file__)))
  File "/usr/lib/python3.7/http/server.py", line 646, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.7/socketserver.py", line 720, in __init__
    self.handle()
  File "/usr/lib/python3.7/http/server.py", line 426, in handle
    self.handle_one_request()
  File "/usr/lib/python3.7/http/server.py", line 414, in handle_one_request
    method()
  File "./ob-watcher.py", line 615, in do_GET
    heading2, mainbody = self.create_fidelity_bond_table(btc_unit)
  File "./ob-watcher.py", line 368, in create_fidelity_bond_table
    + "</tr>"
  File "./ob-watcher.py", line 344, in <lambda>
    elem = lambda e: "<td>" + e + "</td>"
TypeError: can only concatenate str (not "datetime.datetime") to str
```
when clicking the Fidelity Bonds page in the browser.

Reply:
running the ob-watcher.py with the current state `a2e6205` on the 32bit ARM node at:
2ahmbmdgulxzh2qp7u3lvtcvkivfji3o6xvyw4msx7iglopfjrovihad.onion

Reply:
This part would only cause a crash if somebody creates a fidelity bond with a locktime after the year 2038. So I guess its fine to leave it as it is since that's easier than trying to debug right now.

Reply:
Found the issue that the `Redeem script` does not show when run on 32bit ARM, so this line needs to be fixed somehow.
![image](https://user-images.githubusercontent.com/43343391/127854495-cf691cd4-cc43-45f7-af38-349800eb9938.png)

Reply:
ok, the ob-watcher (including the Fidelity Bond tab) on the address above is working now with the current changes.